### ISSUE TITLE
It is not a bug but in the globe demo the earth is spinning backwards.

### ISSUE THREAD

**Issue by mondocosm** [nodes-io/nodes-io#29]
### Environment Information:
- app: 1.0.0-beta.2
- os: darwin
- node: 12.18.3
- electron: 11.1.0
- chrome: 87.0.4280.88

**Comment by vorg** [comment]
That's true! With the current rotation Americas would be waking up before Europe :) Will fix in the next release. Thanks for reporting
